tests/unit/s2n_tls13_keys_test.c

```diff
@@ -167,8 +167,9 @@ int main(int argc, char **argv)
 
     EXPECT_SUCCESS(s2n_tls13_keys_init(&secrets, S2N_HMAC_SHA256));
 
+    struct s2n_psk *psk = NULL;
     /* Derive Early Secrets */
-    EXPECT_SUCCESS(s2n_tls13_derive_early_secrets(&secrets));
+    EXPECT_SUCCESS(s2n_tls13_derive_early_secrets(&secrets, psk));
```

Comment:
Might be cleaner as just `EXPECT_SUCCESS(s2n_tls13_derive_early_secrets(&secrets, NULL));`